No module name encodings

Hey guys :)

I have an issue with LeaderF after updating my macvim:

snapshot-137 works fine,
with snapshot-138 macvim shows:

```
Fatal Python error: Py_Initialize: unable to load the file system codec
ModuleNotFoundError: No module named 'encodings'

Current thread 0x00007fffa668c340 (most recent call first):
Vim: Caught deadly signal ABRT
Vim: Finished.
```

My .vimrc looks super simple

```
" Specify a directory for plugins
" - For Neovim: ~/.local/share/nvim/plugged
" - Avoid using standard Vim directory names like 'plugin'
call plug#begin('~/.vim/plugged')

if empty(glob('~/.vim/autoload/plug.vim'))
  silent !curl -fLo ~/.vim/autoload/plug.vim --create-dirs
    \ https://raw.githubusercontent.com/junegunn/vim-plug/master/plug.vim
  autocmd VimEnter * PlugInstall --sync | source $MYVIMRC
endif

Plug 'Yggdroot/LeaderF', { 'do': './install.sh' }

" Initialize plugin system
call plug#end()
```

I published this issue on LeaderF repo as well: https://github.com/Yggdroot/LeaderF/issues/73

Thanks!

doesn't fix the problem for me, I still get this error:
```
~$ gvim
~$ Fatal Python error: Py_Initialize: unable to load the file system codec
ModuleNotFoundError: No module named 'encodings'

Current thread 0x00007fff9bff2340 (most recent call first):
Vim: Caught deadly signal ABRT
Vim: Finished.
```
Trying to install macvim from source gives me the following error and asks me for access to the keychain:
```
~$ brew reinstall macvim --HEAD
==> Reinstalling macvim 
==> Cloning https://github.com/macvim-dev/macvim.git
Cloning into '/Users/eugencazacu/Library/Caches/Homebrew/macvim--git'...
remote: Counting objects: 3409, done.
remote: Compressing objects: 100% (2884/2884), done.
remote: Total 3409 (delta 530), reused 1563 (delta 425), pack-reused 0
Receiving objects: 100% (3409/3409), 15.24 MiB | 3.27 MiB/s, done.
Resolving deltas: 100% (530/530), done.
==> Checking out branch master
==> ./configure --with-features=huge --enable-multibyte --with-macarchs=x86_64 --enable-perlinterp --enable-rubyinterp --enable-tclinterp -
==> make
Last 15 lines from /Users/eugencazacu/Library/Logs/Homebrew/macvim/02.make:
    export XCODE_DEVELOPER_USR_PATH=/Applications/Xcode.app/Contents/Developer/usr/bin/..
    /Applications/Xcode.app/Contents/Developer/usr/bin/ibtool --strip /tmp/macvim-20180401-4828-1ctlxvt/src/MacVim/build/Release/MacVim.app/Contents/Resources/English.lproj/FindAndReplace.nib --output-format human-readable-text /tmp/macvim-20180401-4828-1ctlxvt/src/MacVim/English.lproj/FindAndReplace.nib

CopyPlistFile build/Release/MacVim.app/Contents/Resources/KeyBinding.plist KeyBinding.plist
    cd /tmp/macvim-20180401-4828-1ctlxvt/src/MacVim
    builtin-copyPlist --outdir /tmp/macvim-20180401-4828-1ctlxvt/src/MacVim/build/Release/MacVim.app/Contents/Resources -- KeyBinding.plist

** BUILD FAILED **


The following build commands failed:
	StripNIB English.lproj/Preferences.nib
(1 failure)
make[1]: *** [macvim] Error 65
make: *** [first] Error 2

READ THIS: https://docs.brew.sh/Troubleshooting

```

--- a/src/MacVim/vimrc
+++ b/src/MacVim/vimrc
@@ -22,15 +22,12 @@ if exists("&pythondll") && exists("&pythonhome")
   if filereadable("/usr/local/Frameworks/Python.framework/Versions/2.7/Python")
     " Homebrew python 2.7
     set pythondll=/usr/local/Frameworks/Python.framework/Versions/2.7/Python
-    set pythonhome=/usr/local/Frameworks/Python.framework/Versions/2.7
   elseif filereadable("/opt/local/Library/Frameworks/Python.framework/Versions/2.7/Python")
     " MacPorts python 2.7
     set pythondll=/opt/local/Library/Frameworks/Python.framework/Versions/2.7/Python
-    set pythonhome=/opt/local/Library/Frameworks/Python.framework/Versions/2.7
   elseif filereadable("/Library/Frameworks/Python.framework/Versions/2.7/Python")
     " https://www.python.org/downloads/mac-osx/
     set pythondll=/Library/Frameworks/Python.framework/Versions/2.7/Python
-    set pythonhome=/Library/Frameworks/Python.framework/Versions/2.7
   endif
 endif
 
@@ -43,11 +40,9 @@ if exists("&pythonthreedll") && exists("&pythonthreehome") &&
   if filereadable("/opt/local/Library/Frameworks/Python.framework/Versions/3.7/Python")
     " MacPorts python 3.7
     set pythonthreedll=/opt/local/Library/Frameworks/Python.framework/Versions/3.7/Python
-    set pythonthreehome=/opt/local/Library/Frameworks/Python.framework/Versions/3.7
   elseif filereadable("/Library/Frameworks/Python.framework/Versions/3.7/Python")
     " https://www.python.org/downloads/mac-osx/
     set pythonthreedll=/Library/Frameworks/Python.framework/Versions/3.7/Python
-    set pythonthreehome=/Library/Frameworks/Python.framework/Versions/3.7
   endif
 endif
 

